lib/discovery.js

```diff
@@ -88,13 +88,13 @@ function mixinDiscovery(MsSQL) {
 
     if (options.all && !owner) {
       sqlTables = paginateSQL('SELECT \'table\' AS "type", table_name AS "name", table_schema AS "owner"' +
-        ' FROM information_schema.tables', 'table_schema, table_name', options);
+        ' FROM INFORMATION_SCHEMA.tables', 'table_schema, table_name', options);
```

Comment:
What about table/column names such as `tables`, `table_schema`, `table_name`? Do we need to make them uppercase too (at least for consistency)?